tests/acl/test_stress_acl.py

```diff
@@ -220,3 +311,251 @@ def test_acl_add_del_stress(rand_selected_dut, tbinfo, ptfadapter, prepare_test_
+        '193.11.192.128/25',
+        '193.11.200.0/25',
+        '193.11.200.128/25',
+        '193.11.208.0/25',
+        '193.11.208.128/25',
+        '193.11.216.0/25',
+        '193.11.216.128/25',
+        '193.11.224.0/25',
+        '193.11.224.128/25',
+        '193.11.232.0/25',
+        '193.11.232.128/25',
+        '193.11.240.0/25',
+        '193.11.240.128/25',
+        '193.11.248.0/25',
+        '193.11.248.128/25'
+    ]
+    rules = {}
+    rules['ACL_RULE'] = {}
+    # /25 subnets have 126 usable IP addresses
+    use_n_subnets = math.ceil(n_rules / 126)
+    src_ip = '20.0.0.1'  # don't care what it is
+    j = 1
+    finish = False
+    action = 'FORWARD'
+    for i in range(use_n_subnets):
+        if finish is True:
+            break
+        subnet = subnets[i]
+        ip_addresses = generate_ipv4_addresses(subnet)
+        for ip in ip_addresses:
+            rule_name = f'{table_name}|RULE_{j}'
+            rules['ACL_RULE'][rule_name] = {
+                'PACKET_ACTION': action,
+                'SRC_IP': src_ip,
+                'DST_IP': ip,
+                'IP_PROTOCOL': '6',
+                'PRIORITY': str(j)
+            }
+            # toggle actions
+            if action == 'FORWARD':
+                action = 'DROP'
+            if action == 'DROP':
+                action = 'FORWARD'
+            if j == n_rules:
+                finish = True
+                break
+            j += 1
+    return rules
+
+@pytest.mark.stress_test
+def test_acl_stress(request, rand_selected_dut, prepare_test_port, tbinfo,  # noqa: F811
+                    ptfadapter, setup_table_and_rules,
+                    get_function_completeness_level, skip_traffic_test):  # noqa: F811
+
+    if skip_traffic_test:
+        return
+
+    if not request.config.getoption("--run-stress-tests"):
+        pytest.skip("Stress tests are skipped. To run them, use --run-stress-tests option.")
+
+    normalized_level = get_function_completeness_level
+    if normalized_level is None:
+        normalized_level = 'debug'
+    loop_times = LOOP_TIMES_LEVEL_MAP[normalized_level]
+
+    logger.debug('Start testing stress acl')
+    ptf_src_port, ptf_dst_ports, dut_port = prepare_test_port
+    logger.debug(f'DUT Port used in test is {dut_port}')
+    acl_rules = setup_table_and_rules
+    logger.debug(f'Number of rules: {len(acl_rules)}')
+    content = None
+    pkt = None
+    loop = 0
+    while loop < loop_times:
+        loop += 1
+        for rule_name, rule in acl_rules.items():
+            if rule.get('IP_PROTOCOL') == '6':
+                # logger.debug('Creating TCP packet')
+                dport = rule.get('L4_DST_PORT') if rule.get('L4_DST_PORT') else '12345'
+                flags = ''
+                fmap = {
+                    'TCP_SYN': 'S',
+                    'TCP_ACK': 'A',
+                    'TCP_URG': 'U',
+                    'TCP_PSH': 'P',
+                    'TCP_RST': 'R',
+                    'TCP_FIN': 'F'
```

Comment:
Why do we need to use different TCP flaps? We don't have rule to match TCP flags.